src/deprecated/support.js

```diff
@@ -0,0 +1,11 @@
+define( [
+	"../var/support"
+], function( support ) {
+
+	"use strict";
+
+	support.trim = "".trim ? "_" === " \uFEFF\xA0_\uFEFF\xA0 ".trim() : false;
+
+	return support;
```

Comment:
We avoid indenting contents of this outer function so that larger files are not completely indented; let's follow that here as well:
```suggestion
"use strict";

support.trim = "".trim ? "_" === " \uFEFF\xA0_\uFEFF\xA0 ".trim() : false;

return support;
```